configs/opencv-ovms/scripts/run_object_detection.sh

```diff
@@ -5,9 +5,6 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-cid_count="${cid_count:=0}"
-inputsrc="${inputsrc:=}"
-
```

Comment:
this here is to set default values in case there is none given,  shellcheck complains it if not giving default values